IN-CORE/incore-playbook
# 421 relax check on duplicate conditions

tests:

Scenario 1:
- select 2 duplicate zone and archetypes
e.g. 
1) 1% zone + archetype 1
2) 1% zone + archetype 1
- select duplicate retrofit method
1) 5 feet
2) 5 feet
- This shouldn't be allowed regardless of the percentage
Should show error. Because below doesnt make sense
- 1% zone + archetype 1  + 5 feet 30%
- 1% zone + archetype 1 + 5 feet 24% 


Scenario 2:
- select duplicate zone and archetypes
1) 1% zone + archetype 1
2) 1% zone + archetype 1
- select different retrofit methods
1) 5 feet
2) 10 feet
- total sum of their percentage > 1 
1) 60%
2) 70%
- This shouldn't be allowed
- 1% zone + archetype 1  + 5 feet 60%
- 1% zone + archetype 1 + 10 feet 70%
